.github/docs/InteractiveMessageFeature.md

```diff
@@ -0,0 +1,439 @@
+            "elements": [
+                {
+                    "title": "Check self-service options"
+                },
+                {
+                    "title": "Talk to an agent"
+                },
+                {
+                    "title": "End chat"
+                }
+            ]
+        }
+    }
+}
+```
+
+Title/subtitle strings can be written with multi-line format, or single line with `\r\n` line break characters.
+
+```js
+const PickerSubtitle = `This is some *emphasized text* and some **strongly emphasized text**
+
+This is a bulleted list (multiline):
+* item 1
+* item 2
+* item 3
+
+This is a numbered list:
+1. item 1
+2. item 2
+3. item 3
+
+Questions? Visit https://plainlink.com/faq
+
+[This is a link](https://aws.amazon.com)
+
+This is \`<code />\`
+`
+const SingleLinePickerSubtitle = "This is some *emphasized text* and some **strongly emphasized text**\r\nThis is a bulleted list:\n* item 1\n* item 2\n* item 3\n\nThis is a numbered list:\n1. item 1\n2. item 2\n3. item 3\n\nQuestions? Visit https://plainlink.com/faq\r\n[This is a link](https://aws.amazon.com)\r\nThis is `<code />`";
+
+const ListPicker = {
+    templateType: "ListPicker",
+    version: "1.0",
+    data: {
+        content: {
+            title: "How can we help? [aws.amazon.com](https://aws.amazon.com)<!--rehype:target=_self-->",
+            subtitle: PickerSubtitle || SingleLinePickerSubtitle,
+            elements: [ /* ... */ ]
+        }
+    }
+}
+```
+
+#### Markdown Links
+
+To create a link, use the following syntax:
+
+```
+[aws](https://aws.amazon.com)
+```
+
+You can also set the link's target attribute using rehype syntax (default is `target="_blank"`):
+
+```
+[plain text](https://example.com)<!--rehype:target=_self-->
+```
+
+### Pagination logic for ListPicker options
+
+Please refer to [`InteractiveMessageActionButtonImplementation.md`](/.github/docs/InteractiveMessageActionButtonImplementation.md)
+
+Currently the max number of list picker options that return from server(via Lex) is 6(10 for panel picker), but our customers would like to provide more options to their end customers. In order to achieve this goal, we have implemented the idea of "Show more" and "Previous options" buttons in our interactive message UI. These changes will require customers to provide some new data when they configure the interactive message so that they are able to use this new feature. [Learn more about setting up action buttons](/.github/docs/InteractiveMessageActionButtonImplementation.md)
+
+<img src="/.github/screenshots/interactive-message-action-buttons.png" width="200px" alt="InteractiveMessageActionButtonScreenshot" />
```

Comment:
This is rendering as a link instead of an inline screenshot.

Reply:
Updated, forgot to add the file to this pull request

// ref: https://github.com/amazon-connect/amazon-connect-chat-interface/blob/8edb2047bd2818b2bc0bb1b2d6266ef4a7ba57c0/screenshots/interactive-message-action-buttons.png